packages/website/bin/index.ts

```diff
@@ -0,0 +1,43 @@
+import {ensureDir, removeSync} from 'fs-extra';
+import * as path from 'node:path';
+import simpleGit, {SimpleGit} from 'simple-git';
+import {DEMO_FILENAME_PATTERN} from './constants';
+import {copyDemoFiles} from './copyDemoFiles';
+import {findTsxAndCssModule} from './findTsxAndCssModule';
+import {generatePage} from './generatePage';
+import {groupFilesByComponent} from './groupFilesByComponent';
+
+const cloneRepository = async (repoUrl: string, destinationFolder: string): Promise<void> => {
+    const git: SimpleGit = simpleGit();
+    await git.clone(repoUrl, destinationFolder);
```

Comment:
Should we add clone to [semantic-monorepo-tools](https://github.com/coveo/semantic-monorepo-tools/tree/main/src/git)?

Reply:
I'll create tasks so we don't forget but I wouldn't block this PR for that if you agree